README_DEPLOYMENT.md

```diff
@@ -0,0 +1,122 @@
+# Geo Browser - Aragon OSx
+
+## Deployment
+
+### On a blockchain supported by Aragon
+
+If the current contracts are meant to be deployed on a blockchain like Mainnet, Sepolia, Polygon, Base, etc, the only step needed is to define the relevant env vars on `.env` and run the following command:
+
+```sh
+cd packages/contracts
+npx hardhat deploy --network <the-network-name>
+```
+
+### On a custom blockchain
+
+In the case of specific blockchains where OSx support is not officially available, you will need to deploy it first. The high level process looks like:
+
+0. Prepare a wallet for the whole deployment process
+1. Deploy OSx
+   1. This will create a Managing DAO, where the deployer wallet can `execute()` actions
+2. Deploy the plugins of this project
+3. Install the plugin that will control the Managing DAO and revoke the `ROOT_PERMISSION` from the deployment wallet
+
+#### 1) Deploy OSx
+
+Clone [the OSx repo](https://github.com/aragon/osx) into a separate folder.
+
+```sh
+yarn
+git checkout simpler-deployment
+cd packages/contracts
+cp .env.example .env
+nano .env    # Update the values
+yarn build
+# yarn test  (optional)
+```
+
+- Edit the `packages/contracts/hardhat.config.ts` to include the details of your target network.
```

Comment:
I don't think this and next line are necessary since `osx`'s deployment explains how to change hardhat.config.ts (in a nutshell, they don't need to change this file at all, but networks.json), so this line and next are not necessary to be here.